jqplot_example/jqplot_example.links.menu.yml

```diff
@@ -0,0 +1,28 @@
+jqplot.example:
+  title: jqplot Example
```

Comment:
My be we should change it to static example as we are going add dynamic examples.